sonarts-core/src/rules/noIdenticalFunctionsRule.ts

```diff
@@ -51,10 +51,12 @@ export class Rule extends tslint.Rules.AbstractRule {
         const originalFunctionBlock = functionBlocks[j];
 
         if (areEquivalent(duplicatingFunctionBlock, originalFunctionBlock)) {
-          visitor.addIssue(
-            Rule.issueNode(duplicatingFunctionBlock.parent as ts.FunctionLikeDeclaration),
-            Rule.message(originalFunctionBlock),
-          );
+          visitor
+            .addIssue(
+              Rule.issueNode(duplicatingFunctionBlock.parent as ts.FunctionLikeDeclaration),
+              Rule.message(originalFunctionBlock),
+            )
+            .addSecondaryLocation(new IssueLocation(originalFunctionBlock.parent!, "original implementation"));
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Forbidden non null assertion ([typescript:S2966](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS2966))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=issue-244&issues=AWCije4x_Q-0dz4JAh0T&open=AWCije4x_Q-0dz4JAh0T)

[ISSUE_KEY]: AWCije4x_Q-0dz4JAh0T

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED